Client/Assets/SearchPlugins/list.json

```diff
@@ -2,646 +2,32 @@
-          "bing", 
-          "google", 
-          "wikipedia-gu", 
-          "yahoo-in", 
-          "eBay-in"
-        ]
-      }
-    }, 
-    "he": {
-      "default": {
-        "visibleDefaultEngines": [
-          "amazondotcom", 
-          "bing", 
-          "google", 
-          "twitter", 
-          "wikipedia-he", 
-          "yahoo"
-        ]
-      }
-    }, 
-    "hi-IN": {
-      "default": {
-        "visibleDefaultEngines": [
-          "amazon-in", 
-          "bing", 
-          "google", 
-          "twitter", 
-          "wikipedia-hi", 
-          "yahoo-in"
-        ]
-      }
-    }, 
-    "hr": {
-      "default": {
-        "visibleDefaultEngines": [
-          "amazon-en-GB", 
-          "bing", 
-          "google", 
-          "twitter", 
-          "wikipedia-hr", 
-          "yahoo"
-        ]
-      }
-    }, 
-    "hsb": {
-      "default": {
-        "visibleDefaultEngines": [
-          "amazon-de", 
-          "bing", 
-          "google", 
-          "twitter", 
-          "wikipedia-hsb", 
-          "yahoo-de"
-        ]
-      }
-    }, 
-    "hu": {
-      "default": {
-        "visibleDefaultEngines": [
-          "google", 
-          "sztaki-en-hu", 
-          "vatera", 
-          "twitter", 
-          "wikipedia-hu"
-        ]
-      }
-    }, 
-    "hy-AM": {
-      "default": {
-        "visibleDefaultEngines": [
-          "amazondotcom", 
-          "bing", 
-          "google", 
-          "list-am", 
-          "wikipedia-hy-AM", 
-          "yahoo"
-        ]
-      }
-    }, 
-    "ia": {
-      "default": {
-        "visibleDefaultEngines": [
-          "google", 
-          "bing", 
-          "amazondotcom", 
-          "twitter", 
-          "wikipedia-ia"
-        ]
-      }
-    }, 
-    "id": {
-      "default": {
-        "visibleDefaultEngines": [
-          "bing", 
-          "google", 
-          "twitter", 
-          "wikipedia-id", 
-          "yahoo-id"
-        ]
-      }
-    }, 
-    "is": {
-      "default": {
-        "visibleDefaultEngines": [
-          "amazondotcom", 
-          "bing", 
-          "google", 
-          "leit-is", 
-          "wikipedia-is", 
-          "yahoo"
-        ]
-      }
-    }, 
-    "it": {
-      "default": {
-        "visibleDefaultEngines": [
-          "google", 
-          "twitter", 
-          "wikipedia-it", 
-          "bing", 
-          "eBay-it"
-        ]
-      }
-    }, 
-    "ja": {
-      "default": {
-        "visibleDefaultEngines": [
-          "amazon-jp", 
-          "bing", 
-          "google", 
-          "twitter-ja", 
-          "wikipedia-ja", 
-          "yahoo-jp"
-        ]
-      }
-    }, 
-    "ka": {
+  "locales": {  
+    "en": {
+      "searchDefault": "bing",
       "default": {
         "visibleDefaultEngines": [
-          "amazondotcom", 
           "bing", 
-          "google", 
-          "wikipedia-ka", 
-          "yahoo", 
-          "twitter"
+          "yandex"
```

Comment:
yandex is not on the list here: https://github.com/brave/brave-core/pull/1648

Reply:
https://bravesoftware.slack.com/archives/CELA35G5S/p1550094658036400?thread_ts=1550020220.000600&cid=CELA35G5S
Coming from here, but will gladly remove if needed.

Reply:
Also Yandex is showing translated text for EN as well. Do we need to have this fixed or is it ok?
![image](https://user-images.githubusercontent.com/17010094/53076994-096a6b80-3517-11e9-80c4-0247732b879f.png)